english/vote-the-president/hillary/Hillary.md

```diff
@@ -1,3 +1,5 @@
 Congratulations. You made another Clinton president again. 99% of Hillary voters are actually Trump haters.
 
-You have done God's work. [#NeverTrump](https://twitter.com/hashtag/NeverTrump?src=hash) 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.